bcipy/gui/intertask_action.py

```diff
@@ -0,0 +1,57 @@
+from bcipy.gui.bciui import BCIUI, run_bciui
+from PyQt6.QtWidgets import (
+    QApplication,
+    QLabel,
+    QHBoxLayout,
+    QPushButton,
+    QProgressBar,
+)
+
+
+class IntertaskAction(BCIUI):
```

Comment:
This should likely be InterTask GUI, the Action itself will need to inherit Task.